server/routes/index.ts

```diff
@@ -35,7 +36,7 @@ export default function routes(router: Router, services: Services): Router {
   get('/integrations/oasys/assessment', integrationSamples.viewOasysAssessmentSample)
 
   get('/referrals/start', (req, res) => referralsController.startReferral(req, res))
-  post('/referrals', (req, res) => referralsController.createReferral(req, res))
+  post('/referrals/start', (req, res) => referralsController.createReferral(req, res))
```

Comment:
changed this to avoid inconsistency when you submit the start referral form with an error and the url changes. but there's weird "back" button behaviour that i don't like